examples/twittermap/web/conf/application.conf

```diff
@@ -90,7 +90,7 @@ predefinedKeywords = ["Diabetes", "Sleep Disorder"]
 startDate = "2015-11-22T00:00:00.000"
 
 # Set the endDate if the data is stopped feeding
-# endDate = "2017-06-20T00:00:00.000"
+endDate = "2017-05-01T00:00:00.000"
```

Comment:
The `Drum` slicing doesn't allow jump too far to avoid the potential risk of missing a deadline. 
Internally, it doesn't allow the next time range grows twice bigger than the current one. 
The problem of this limit is that the current develop sample twitter demo only contains the data from 2015 to 2017-05-01. Without this `endDate`, the slicing is from `now`, then the update is too slow. By correctly specifying the `endDate`, the slicing is from `2017-05-01`, then developers will see the result much faster.

Reply:
Got it.

Reply:
Got it.